res/values-de/strings.xml

```diff
@@ -9,8 +9,8 @@
     <string name="Button_Margin_">"Schaltflächen Position:"</string>
     <string name="Buttons_">"Schaltflächen:"</string>
     <string name="DEFAULTS">"Vorgaben"</string>
-    <string name="WEEKLY">"Wöchentlich"</string>
-    <string name="DAILY">"Täglich"</string>
+    <string name="WEEKLY">"wöchentlich"</string>
+    <string name="DAILY">"täglich"</string>
```

Comment:
Lines 12 and 13: Why removed the capitals? Btw, this was actually supposed to be 100% capirals like in `res/values`.

Reply:
It was lowercase before too, I see in other language xml files it's always uppercase. It doesn't really matter, though, because those are the translations for the leaderboard, which is always set to all uppercase by the game.

I reverted those changes since those words aren't "Nomen" and should be lowercase for German sentences.

Reply:
In Android, the fact that it's uppercase is true, but you must think about the iOS users too.

Reply:
I updated the commit, it's now uppercase